src/OrderLinesList/constants.js

```diff
@@ -39,3 +39,10 @@ export const BOOLEAN_OPTIONS = [
     label: <FormattedMessage id="ui-orders.filter.false" />,
   },
 ];
+
+export const sources = {
+  user: 'User',
```

Comment:
I believe it will be reused across all Acq modules, so it's better to put it under stripes-acq-components